src/types.ts

```diff
@@ -56,6 +56,18 @@ export enum IconColorType {
     CONTRAST = 'contrast',
 }
 
+export interface CustomTile {
```

Comment:
Om du har kun denne felles typen, men innfører eit felt som kan skille på QR og Image, så er det lettare å skille på desse enn å bruke `displayHeader in`. Bruk gjerne `TileType`, og gjerne som `enum` som Caroline foreslo:
```
enum TileType {
  QR = 'QR',
  IMAGE = 'IMAGE'
}

export interface CustomTile {
  type: TileType,
  id: string,
  displayName: string
  linkAddress: string
  description?: string
  displayHeader?: string
}
```

Det er vel ikkje heilt utenkeleg at QR-tiles kan ha støtte for displayHeader, så det kan kanskje vere eit optional felt. `description` bør kanskje òg vere optional?